Method to use IAM roles (#3017)
astronomer/docs: software/registry-backend.md

3. Push the configuration change to your platform. See [Apply a config change](apply-platform-config.md).

> ### Method to use IAM roles  

Reviewer: ```suggestion
### Use IAM roles (Optional)
```

> Add the following values to your `config.yaml`to use IAM roles:

Reviewer: ```suggestion
To use an IAM role instead of an IAM user to authenticate to your registry backend, add the following configuration to your `config.yaml` file:
```

Reviewer 2: This feature replaces the use of hardcoded credentials

> ```

Reviewer: ```suggestion
```yaml
```

registry: 
    serviceAccount: 
      # Specifies whether a service account should be created 
      create: true 
      # Annotations to add to the service account 
      annotations: 

>         eks.amazonaws.com/role-arn: arn:aws:iam::xxxxxxxxxxxxxx:role/srecluster100-registry-backend-role 

Reviewer: ```suggestion
        eks.amazonaws.com/role-arn: arn:aws:iam::xxxxxxxxxxxxxx:role/<your-iam-role>
```

    s3: 
      enabled: true 

>       region: us-east-2 

Reviewer: ```suggestion
      region: <your-region>
```

>       bucket: srecluster100-registry-backend

Reviewer: ```suggestion
      bucket: <your-registry-backend>
```

```

> 

Reviewer: ```suggestion

Then, push the configuration change to your platform. See [Apply a config change](apply-platform-config.md).
```
